eda-env/Lib/site-packages/parso/pgen2/grammar_parser.py

```
@@ -0,0 +1,160 @@
+        self._gettoken()
+        return value
+
+    def _gettoken(self):
+        tup = next(self.generator)
+        self.type, self.value, self.begin, prefix = tup
+
+    def _raise_error(self, msg, *args):
+        if args:
+            try:
+                msg = msg % args
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)